.github/workflows/official-images.yml

```diff
@@ -0,0 +1,63 @@
+name: Official Images Updates
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "master"
+      - "8"
+      - "7"
+      - "6"
+  pull_request:
+
+permissions: {}
+
+jobs:
+  library-gradle:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0 # we need all branches so we can read Dockerfiles from them
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - name: Provision best available generate-stackbrew-library.sh
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+          CURRENT_COMMIT: ${{ github.sha }}
+          BASE_REF: ${{ github.event.pull_request.base.ref }}
+        # if the script was modified in the current PR, use that version; otherwise, use the version from master
+        shell: bash
+        run: |
+          if [ "${EVENT_NAME}" == "pull_request" ] && [ "$(git diff --name-only "origin/$BASE_REF..$CURRENT_COMMIT" -- 'generate-stackbrew-library.sh' | wc -l)" -gt 0 ]; then
+            echo "Using modified generate-stackbrew-library.sh"
+          else
+            echo "Using generate-stackbrew-library.sh from master branch"
+            git checkout origin/master -- generate-stackbrew-library.sh
+          fi
+      - name: Generate library file
+        uses: docker://bash:latest
+        with:
+          # See https://github.com/docker-library/bashbrew?tab=readme-ov-file#installing
+          args: |
+            -c "\
+              apk add --no-cache jq git curl ca-certificates && \
+              git config --global --add safe.directory '*' && \
+              curl -fsSL -o /usr/local/bin/bashbrew https://doi-janky.infosiftr.net/job/bashbrew/job/master/lastSuccessfulBuild/artifact/bashbrew-amd64 && \
```

Comment:
🤔 Should we download that file from there or from another place _we trust_?
Is there at least some service/info where we could check the integrity of what we get? Like signature, checksum, or ...